nats-init

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
```

Comment:
Move this script somewhere under `etc`

Reply:
Would be best under node_ws, but need that branch to be pulled first. Let me know what you think.

Reply:
Git only versions files, so you can add the folder now

Reply:
Done in node-ws branch